ui/app/components/app/confirmation-footer/confirmation-footer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Button from '../../ui/button';
+
+export default function ConfirmationFooter({
+  onApprove,
+  onCancel,
+  approveText,
+  cancelText,
+  alerts,
+}) {
+  const className = classnames('confirmation-footer', {
+    'confirmation-footer--with-alerts':
+      alerts && Object.keys(alerts).length > 0,
+  });
+
+  return (
+    <div className={className}>
+      {alerts && alerts}
+      <div className="confirmation-footer__actions">
+        <Button
+          rounded
+          type="secondary"
+          onClick={onCancel}
+          onKeyUp={(event) => {
+            if (event.key === 'Enter') {
```

Comment:
This is for accessibility, correct? Not for this PR, but in the future we could build this functionality into the button component, so that we don't need to add an `onKeyUp` property to every button